### DISCUSSION TITLE
xDrip with G6 reading data once every 20-25 minutes

### DISCUSSION THREAD

**Discussion by gyusunglee** [NightscoutFoundation/xDrip#2147] [General]
Hi,

I am using Samsung S22 Ultra (Android 12) with xDrip 2022.01.31 version. After a factory reset of my phone, my xDip reads Dexcom G6 data not every 5 minutes, but every 20-25 minutes. I am using all the recommended settings, except for the "Allow OB1 unbending" option (currently unchecked). Any suggestions for fixing this problem? Thanks
![Screenshot_20220527-230726_xDrip+](https://user-images.githubusercontent.com/50647359/170809578-b2f79d9c-eae9-4f05-b653-06110c79825a.jpg)

![Screenshot_20220527-230800_xDrip+](https://user-images.githubusercontent.com/50647359/170809585-70814881-9535-42fb-9fdb-e7b64f804006.jpg)

.

**Comment by Navid200** [comment]
Can you post a screenshot of the system status page?  You can blackout the transmitter serial number if you like.

**Reply by gyusunglee** [reply]
I added several screenshots. I hope this will help you. Thanks.

**Reply by Navid200** [reply]
You have revealed your Nightscout ID.  The problem is with the master, not the follower, right?
If the problem is with the master, you can delete the second screenshot.  Or, you should blackout the ID.

**Comment by gyusunglee** [comment]
Hi, Navid,

 

I added a few screenshots to the post. Thanks.

 

Gyusung

**Comment by Navid200** [comment]
Are you sure the battery optimization is disabled for xDrip as well as for Bluetooth?

**Comment by gyusunglee** [comment]
Yes. It was with the master. I deleted the screenshot. Thanks.

**Comment by gyusunglee** [comment]
Yes. Disabled for both.

**Comment by Navid200** [comment]
Do you see "Last Connected" on your system status page?
Does that go over 5 minutes when there is a delay?

**Reply by Navid200** [reply]
Please answer this question also.

**Comment by Navid200** [comment]
Do you have another phone that also collects from the same transmitter either with xDrip or Dexcom app?

**Comment by gyusunglee** [comment]
No. Only one phone with xDrip app. I use a Dexcom receiver, too.

**Comment by gyusunglee** [comment]
Hi Navid,

 

When I took the screenshot, I had no reading for the last 16 minutes, but the system status showed that the last connection had occurred 1 minute ago. 

 

For some reason, it seems to be working better today. Since I was outside without my phone to do some gardening, I will see what happens tomorrow. If I have the same problem again, I will check the system status again and let you know. Thanks.

 

Gyusung

**Comment by Navid200** [comment]
Please try the latest Nightly if you are still having connectivity problems:
https://github.com/NightscoutFoundation/xDrip/releases/tag/2022.06.09

**Comment by gyusunglee** [comment]
Hi Navid. 

 

Thanks. The connectivity issue has recently been minimized. I only noticed occasionally. However, I just installed the newest version. Should I have the “Allow OB1 unbonding” checked or unchecked? Thanks for your effort with this life-saving program.

**Reply by Navid200** [reply]
To being with, please enable "Allow OB1 unbonding".

If the problem returns, please disable it.

Please report either way what you experience.

Thanks

**Comment by gyusunglee** [comment]
OK. Thanks. I will.
